src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -98,24 +98,26 @@ class ConnectionHandlerTest : public ::testing::Test {
 
   void SetSpecificServices() {
 #ifdef ENABLE_SECURITY
-    ON_CALL(mock_connection_handler_settings, force_protected_service())
+    ON_CALL(mock_connection_handler_settings_, force_protected_service())
         .WillByDefault(ReturnRefOfCopy(protected_services_));
-    ON_CALL(mock_connection_handler_settings, force_unprotected_service())
+    ON_CALL(mock_connection_handler_settings_, force_unprotected_service())
         .WillByDefault(ReturnRefOfCopy(unprotected_services_));
 #endif  // ENABLE_SECURITY
   }
+
   // Additional SetUp
   void AddTestDeviceConnection() {
     const transport_manager::DeviceInfo device_info(
         device_handle_, mac_address_, device_name_, connection_type_);
     // Add Device and connection
-    ON_CALL(mock_connection_handler_settings, heart_beat_timeout())
+    ON_CALL(mock_connection_handler_settings_, heart_beat_timeout())
         .WillByDefault(Return(1000u));
     connection_handler_->addDeviceConnection(device_info, uid_);
     connection_key_ = connection_handler_->KeyFromPair(uid_, 0u);
     // Remove all specific services
   }
-  void AddTestSession() {
+
+  uint32_t AddTestSession() {
```

Comment:
@AByzhynar Provide description please

Reply:
@LuxoftAKutsan Done in da7e745e88ac17a2efe416236222f1887549cbde